environs/suite_test.go

```diff
@@ -9,6 +9,8 @@ import (
 	"github.com/juju/juju/testing"
 )
 
+//go:generate mockgen -package mocks -destination mocks/package_mock.go github.com/juju/juju/environs NetworkingEnviron,CloudEnvironProvider
```

Comment:
Nit: can we rename this file to `package_test.go` so it's consistent with everything else.